README.md

```diff
@@ -50,6 +54,96 @@ graph TD
     G -- Results saved to S3<br>with new run ID --> I
 
 ```
+#### Glue Job
+And here we can see how the recurrent glue job will process unflagged sales:
+
+```mermaid
+
+graph TD
+
+    A[Schedule for glue job triggers a run]
+    B{{Ingest data needed to flag unflagged sales}}
+    C[Run flagging model within glue job]
+
+    D[Write sales data to sale.flag]
+    E[Flags joined to<br>default.vw_pin_sale]
+
+    A -- Some sales are not flagged --> B
+    B --> C
+    C --> D
+    D -- Results saved to S3<br>with unique run ID --> E
+
+
+```
+
+# Flags at a Glance
+
+Sales from 2014 - present have been processed using our sales validation program. We assign flags to the following classes of properties:
```

Comment:
**issue:** Add note about not flagging certain residential sales (multi-PIN, multi-card, etc.).